src/audits/DataTableHeadersMissing.js

```diff
@@ -0,0 +1,81 @@
+// Copyright 2015 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+goog.require('axs.AuditRules');
+goog.require('axs.browserUtils');
+goog.require('axs.constants.Severity');
+
+(function () {
+    function tableDoesNotHaveHeaderRow(rows) {
```

Comment:
Yeah, my preference is for things to go in axs.utils unless it really can only apply to the current test, since things might be useful for the debugging UI in the extension. (In fact, https://github.com/GoogleChrome/accessibility-developer-tools/blob/master/src/audits/RequiredOwnedAriaRoleMissing.js#L75 should probably move into axs.utils as well at some point.)

That said, perhaps these methods are single purpose enough that they can stay here for the time being - could you just JSDoc them instead?